openshift-ci/build-root/Dockerfile

```diff
@@ -2,4 +2,27 @@
 
 FROM registry.ci.openshift.org/openshift/release:golang-1.18
 
-RUN yum -y install dnf httpd-tools
+ARG KUBECTL_KUTTL_VERSION=0.12.1
+ARG OPERATOR_SDK_VERSION=1.22.2
```

Comment:
Can you update the version to v1.11.0?
https://github.com/redhat-developer/gitops-operator/pull/526/files

Reply:
Operator SDK version?